Worktrolly/worktrolly
# Dev/task bug fix

### Functionality:
Bug in deleteTask api that overwrites the new tasks created after deleting an existing task.

### Solution:
By changing Category to "Trash" and SprintNumber to '-2' , overwriting of the tasks is avoided. 

### Risk level:

### How to test:
1. ng serve
2. Create 2 tasks under the same category
3. Delete the first task and create a third task
4. You will notice that the data of task 2 is not overwritten by task 3 details.
